src/saving/SaveHelper.cs

```diff
@@ -109,8 +109,11 @@ public static void AutoSave(MicrobeEditor state)
     /// <param name="name">The name of the save to load</param>
     public static void LoadSave(string name)
     {
-        GD.Print("Starting load of save: ", name);
-        new InProgressLoad(name).Start();
+        if (!InProgressLoad.IsLoading)
+        {
+            GD.Print("Starting load of save: ", name);
+            new InProgressLoad(name).Start();
+        }
```

Comment:
Move this check to quick load handler. I still don't want to completely lock the player out from loading a save from manually.
Also add an else with a print to say why it wasn't loaded.